test/dist_autograd_test.py

```diff
@@ -697,29 +697,7 @@ def test_rpc_complex_args(self):
     def test_remote_complex_args(self):
         self._test_rpc_complex_args(ExecMode.REMOTE)
 
-    @dist_init
-    def test_context_cleanup_tensor_with_grad(self):
-        self._initialize_pg()
-
-        dst_ranks = {rank for rank in range(self.world_size) if rank != self.rank}
-        with dist_autograd.context() as context_id:
-            t1 = torch.ones(3, 3, requires_grad=True)
-            t2 = torch.zeros(3, 3, requires_grad=True)
-            for dst_rank in dst_ranks:
-                ret = rpc.rpc_sync("worker{}".format(dst_rank), torch.add, args=(t1, t2))
-                rpc.rpc_sync("worker{}".format(dst_rank), _set_rpc_done, args=(context_id, 1))
-        # the thread's context id should be cleaned up
-        with self.assertRaises(RuntimeError):
-            dist_autograd._retrieve_context(context_id)
-        # Ensure all peers have finished mutating the
-        # `known_context_ids` set.
-        dist.barrier()
-        # check that all contexts have been cleaned up.
-        success = _all_contexts_cleaned_up()
-        self.assertTrue(success)
-
-    @dist_init
-    def test_context_cleanup_tensor_no_grad(self):
+    def context_cleanup_test_helper(self, rpcargs, func):
```

Comment:
nit: use `camel_case` for `rpcargs`.

Reply:
I believe you meant `snake_case` - changed accordingly.